migration/20190215-update-enki-collection-integration.sql

```diff
@@ -0,0 +1,18 @@
+-- Moving the "Library ID" Enki integration setting to be associated with each library
+-- and not just the collection.
+update configurationsettings as cs
+set
+    external_integration_id=updatesetting.externalintegration_id,
+    library_id=updatesetting.library_id,
+    key='external_account_id',
+    value=updatesetting.external_account_id
+from (
+    select externalintegration_id, library_id, external_account_id
+    from collections join externalintegrations_libraries as el
+    on collections.external_integration_id=el.externalintegration_id
+    join externalintegrations as e on e.id=el.externalintegration_id where e.protocol='Enki'
+) updatesetting
+where
+    cs.external_integration_id=updatesetting.externalintegration_id
+    and cs.library_id=updatesetting.library_id
+    and key='external_account_id';
```

Comment:
This is pretty close but it needs to be an `insert` statement rather than an `update`. We're going from a situation where there are no Enki library IDs in `configurationsettings` -- they're all stored in `collections.external_account_id` -- to a situation where there is one Enki library ID in `configurationsetting` for every Enki collection. Basically move the value from `collections.external_account_id` to a new row in `configurationsettings`.